util/envDynawo.sh

```diff
@@ -671,14 +679,33 @@ config_3rd_party() {
 
 build_3rd_party() {
   config_3rd_party || error_exit "Error with 3rd parties configuration."
-  make -j $DYNAWO_NB_PROCESSORS_USED
+  if [ "$DYNAWO_CMAKE_GENERATOR" = "Unix Makefiles" ]; then
```

Comment:
Please keep the behavior by default that builds everything

Reply:
It is, if you give nothing it builds everything